Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+
+Cyber security is involved from the beginning of IT solution building to the end, and relies on as much automated testing as possible to provide assurance, as opposed to gate-based assessment processes, and relies on embedded knowledge within development teams instead of analysts.
+
+Cyber security manages risks differently depending on which network particular work is being done (production, corporate network, or the Internet) in order to focus efforts more effectively and not hinder operations or innovation.
+
+IT at ESDC adopts modern and agile approaches to development to enable greater adaptability to evolutions in both business requirements and enabling technologies.
+IT solution changes are delivered to production within four months, at most, of a change request.
+ESDC will move towards a micro-services pattern when delivering IT solutions, allowing for greater interoperability and smaller, cheaper, more rapid changes.
+Only through well justified and documented exceptions will IT solutions be allowed to remain as a monolith system.
+
+## 5. A digitally enabled public service
+
+Modern web services (SaaS accessible over the Internet) are opened by default and only on rare occasions, with sound justifications, are they blocked.
+Monitoring of web services is risk managed to focus efforts where risks are perceived and impacts are high.
+If monitoring is required, automation through a combination of Artificial Intelligence disciplines is used to reduce toil.
+
+ESDC staff is highly connected and mobile.
+This includes service canada representatives that can answer citizen requests from anywhere, anytime (when AI disciplines cannot complete the request).
+To work at ESDC, one does not necessarily require government managed devices.
+ESDC staff are allowed to bring their own device to perform their work and, should they chose not to or cannot afford one, are provided with a managed mobile device.
+Cyber Security is built-in ESDC's systems so that whether it be an unmanaged device or a managed one that connects to its systems, the assurance level sought is not tied to the device connecting to ESDC's networks but rather to ESDC's cyber security posture and resilience.
+
+Information records have changed to data which is more granular and continually changing.
+Documents are rare, records management systems are obsolete.
+Efficient digital processes have rendered paper records obsolete. This means:
+
+- Briefing notes are digitized
+- Corporate services, such as HR and Finance, do not require ESDC staff to provide information on their first day of work as the hiring process already have that information
+
+Instead the information used by ESDC's staff is presented online via a User Experience that seamlessly integrates the combination of data from multiple sources.
+
+ESDC, with partnership with the union, has changed IT talent management for the federal government to improve recruitment, retention of talent, and wellbeing by:
```

Comment:
- "has changed IT talent management for the federal government" - Isn't that a bit ambitious?
- "with partnership" = "in partnership" I believe?